lib/ansible/modules/network/fortios/fortios_ipv4_policy.py

```diff
@@ -0,0 +1,324 @@
+    description:
+      - Negate source address param.
+    default: false
+    choices: ["true", "false"]
+  dst_addr:
+    description:
+      - Specifies destination address (or group) object name(s).
+    required: true
+  dst_addr_negate:
+    description:
+      - Negate destination address param.
+    default: false
+    choices: ["true", "false"]
+  action:
+    description:
+      - Specifies accept or deny action policy.
+    choices: ['accept', 'deny']
+    required: true
+  service:
+    description:
+      - "Specifies policy service(s), could be a list (ex: ['MAIL','DNS'])."
+    required: true
+    aliases:
+      - services
+  service_negate:
+    description:
+      - Negate policy service(s) defined in service value.
+    default: false
+    choices: ["true", "false"]
+  schedule:
+    description:
+      - defines policy schedule.
+    default: 'always'
+  nat:
+    description:
+      - Enable or disable Nat.
+    default: false
+    choices: ["true", "false"]
+  fixedport:
+    description:
+      - Use fixed port for nat.
+    default: false
+    choices: ["true", "false"]
+  poolname:
+    description:
+      - Specifies nat pool name.
+  av_profile:
+    description:
+      - Specifies Antivirus profile name.
+  webfilter_profile:
+    description:
+      - Specifies Webfilter profile name.
+  ips_sensor:
+    description:
+      - Specifies IPS Sensor profile name.
+  application_list:
+    description:
+      - Specifies Application Control name.
+  comment:
+    description:
+      - free text to describe policy.
+notes:
+  - This module requires pyFG library.
+"""
+
+EXAMPLES = """
+- name: Allow external DNS call
+  fortios_ipv4_policy:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    id: 42
+    srcaddr: internal_network
+    dstaddr: any
+    service: dns
+    nat: True
+    state: present
+    action: accept
+
+- name: Public Web
+  fortios_ipv4_policy:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    id: 42
+    srcaddr: any
+    dstaddr: webservers
+    services:
+      - http
+      - https
+    state: present
+    action: accept
+"""
+
+RETURN = """
+firewall_address_config:
+  description: full firewall adresses config string
+  returned: always
+  type: string
+change_string:
+  description: The commands executed by the module
+  returned: only if config changed
+  type: string
+"""
+
+from ansible.module_utils.fortios import fortios_argument_spec, fortios_required_if
+from ansible.module_utils.fortios import backup
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.pycompat24 import get_exception
+
+#check for pyFG lib
+try:
+    from pyFG import FortiOS, FortiConfig
+    from pyFG.fortios import logger
+    from pyFG.exceptions import CommandExecutionException, FailedCommit, ForcedCommit
+    HAS_PYFG=True
+except:
+    HAS_PYFG=False
+
+def main():
+    argument_spec = dict(
+        comment                   = dict(),
```

Comment:
The default type is `str` but it could be added for the `comment` parameter in order te be consistent with other `str` parameter declaration (`poolname`, `av_profile`, `webfilter_profile` ...).